core/envelope-sim/src/test/java/fr/sncf/osrd/envelope/EnvelopeBuilderTest.java

```diff
@@ -14,6 +14,16 @@ public void testEnvelopeBuilder() {
         assertTrue(env.continuous);
     }
 
+    @Test
+    public void testEnvelopeBuilderEpsilonSpeedDiscontinuity() {
```

Comment:
You're testing the continuity here: `testEnvelopeBuilderEpsilonSpeedDiscontinuity`? And I'd add another test (or test case if you wanna go for a parametrised test) to show the discontinuity as soon as the difference is equal to 1e-5.

Reply:
Thanks, you're right.
Done (amended commit): https://github.com/OpenRailAssociation/osrd/compare/9f35aa419bfef92a3946061fd03defbd8aeee5fc..1310a4d443341c9da08f67318b22e3f5eb856422